src/upgradeDB.js

```
@@ -295,6 +296,46 @@
+          return user.save();
+        });
+
+        await Promise.all(userPromises);
+
+        logger.info('Successfully updated user groups based on new role model');
+        resolve();
+      } catch (err) {
+        logger.error(`Error updating user groups: ${err}`);
+        reject(err);
+      }
+    });
```

Biome

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)